sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -32,6 +32,8 @@ class GestureArenaEntry {
 class GestureArena {
```

Comment:
Document what GestureArenaMember and GestureArenaEntry are (if i understand correctly, the Member is the interface that you have to implement to participate in an Arena, and the Entry is the handle you get back from being in the Arena, and there's an many-to-one relationship, since one Member can be in the Arena multiple times with multiple Entrys).